packages/docs/site/docs/main/contributing/code.md

```diff
@@ -80,3 +80,82 @@ If you're using VS Code and have Chrome installed, you can debug Playground in t
 ### Debugging PHP
 
 Playground logs PHP errors in the browser console after every PHP request.
+
+## Publishing packages
+
+Playground consists of a number of packages, some of which are published to npmjs.com, under the `@wp-playground/` _organization_. While packages are normally automatically published through a [GitHub Action](https://github.com/WordPress/wordpress-playground/actions/workflows/publish-npm-packages.yml), it's also possible to do so from a local machine by running the same script that the GitHub Action runs.
+
+Additionally, it's possible to test-publish packages to a local registry, so that changes can be tested without publishing the package to npmjs.com.
+
+The release process is automated through [lerna](https://lerna.js.org).
+
+### Versioning strategy
+
+Playground's versioning strategy is to use the same version for all packages, **but** only packages that need to be released are bumped to the new version. As an example, lets consider the following scenario:
+
+-   All packages are currently at `v1.0.0`
+-   The following packages have changes since `v1.0.0`: `@wp-playground/cli` and `@wp-playground/remote`
+
+When we issue a new release, only `@wp-playground/cli` and `@wp-playground/remote` will be bumped to `v1.0.1`, and all other packages will remain at `v1.0.0`.
+
+### Authenticating with npmjs.com
+
+TODO
```

Comment:
I think I've heard someone (maybe @bgrgicak ?) mention that they were able to publish to npmjs.com from their local machine, but I'm not 100% sure of this. If you have done it before, could you comment here on how that process exactly works? Thanks in advance. I will then incorporate it into the docs.

Reply:
We have a `npm run release` command that you can run locally. AFAIR it will ask you for npmjs.org credentials.